spec/queries/admin/gdpr_delete_requests_query_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe Admin::GDPRDeleteRequestsQuery, type: :query do
+  subject { described_class.call(search: search) }
+
+  let!(:gdpr_delete_request_1) { create(:gdpr_delete_request, username: "delete_1", email: "delete_1@test.com") }
+  let!(:gdpr_delete_request_2) { create(:gdpr_delete_request, username: "delete_2", email: "delete_2@test.com") }
+  let!(:gdpr_delete_request_3) { create(:gdpr_delete_request, username: "delete_3", email: "delete_3@test.com") }
+  let!(:gdpr_delete_request_11) { create(:gdpr_delete_request, username: "delete_11", email: "delete_11@test.com") }
+
+  describe ".call" do
+    context "when no arguments are given" do
+      it "returns all users" do
+        # rubocop:disable Layout/LineLength
+        expect(described_class.call).to match_array([gdpr_delete_request_1, gdpr_delete_request_2, gdpr_delete_request_3, gdpr_delete_request_11])
+        # rubocop:enable Layout/LineLength
+      end
+    end
+
+    context "when searching for a user by username" do
+      let(:search) { { search: "delete_1" } }
+
+      it { is_expected.to match_array([gdpr_delete_request_1]) }
```

Comment:
```suggestion
      let(:search) { { search: "delete_2" } }

      it { is_expected.to match_array([gdpr_delete_request_2]) }
```

Reply:
Changed this to `delete_2` since `delete_1` will account for both `delete_1` and `delete_11`.